CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -520,8 +497,7 @@ import Vision
         
         let rect = CGRect(x: cx - width / 2.0, y: cy - height / 2.0, width: width, height: height)
         
-        self.currentImageRect = rect
```

Comment:
what is this change for?

Reply:
this is from one of my first commits, i just got rid of it since it doesnt looks like we're using it anywhere

Reply:
Let's just leave it in for now, we can do another PR that removes this that's separate.